### PR TITLE
Feature/symbols by stock market

### PR DESCRIPTION
**Pull request by Waasi**
Implementing Symbols by Stock Market

  - Implemented the FinanceUtils Module
    - Implemented MARKETS variable that has stock markets url by name and country
    - Implemented a helper function that gets an array of symbols by market using MARKETS
    - Refactored YahooFinance Module to have a Client class
    - Refactored Tests to use the Client instead
    - Implemented tests for the FinanceUtils Module

-

- [x] Ready For Review

### CODE REVIEW

File: lib/yahoo-finance/version.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Version Variable for Yahoo Finance Gem
+module YahooFinance
+  VERSION = '0.3.0'
```
Comment (herval) [inline lib/yahoo-finance/version.rb]: might want to change this to 1.0 with all these changes :)

File: yahoo-finance.gemspec [hunk 2]
```diff
@@ -1,16 +1,17 @@
-$LOAD_PATH << '.'
-require 'lib/yahoo_finance'
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'yahoo-finance/version'
 
 spec = Gem::Specification.new do |s|
-  s.name = "yahoo-finance"
+  s.name = 'yahoo-finance'
```
Comment (herval) [inline yahoo-finance.gemspec]: what's with all the single quotes? Can you keep them as double-quotes, as before?

**Comment by herval** [comment]
other than small nitpicks, looking good!

File: yahoo-finance.gemspec (hunk 2 above)
Comment (Waasi) [inline yahoo-finance.gemspec]: Its just that I prefer them but I can change it back.

Comment (herval) [inline yahoo-finance.gemspec]: I think the diff would be smaller
double-quotes also allow string interpolation, so all in all I think using just them is more consistent

Comment (Waasi) [inline yahoo-finance.gemspec]: I agree. I just use a ruby linter and it tells me to use single quotes when string interpolation is not needed. So I am used to that.

**Pull request merged**